ark/utils/spatial_analysis_utils.py

```diff
@@ -248,7 +245,7 @@ def compute_close_cell_num_random(marker_nums, dist_mat, dist_lim, bootstrap_num
     """
 
     # Generate binarized distance matrix
-    dist_mat_bin = (dist_mat.values < dist_lim).astype(np.uint16)
+    dist_mat_bin = ((dist_mat.values < dist_lim) & (dist_mat.values > 0)).astype(np.uint16)
```

Comment:
This greater than 0 is to avoid self-counting right?

Reply:
yep yep. I've gotten word from Inna and Erin that self-counting is officially a no-no (which is strongly represented in the enrichment z-scores as well).  

As for this method, I suppose there's theoretically a case where two cells share a centroid, but I doubt that ever happens in practice.  If you think it's a concern, I can just zero the diagonals after the initial binarization.